.github/workflows/e2e.yml

```diff
@@ -0,0 +1,40 @@
+name: ContractClient
+
+on:
+  push:
+    branches: [master, release/**]
+  pull_request:
+
+jobs:
+  test:
+    name: test generated ContractClient
+    runs-on: ubuntu-22.04
+    services:
+      rpc:
+        image: stellar/quickstart:testing@sha256:6374a596eac0c7b346c162e79dc5ffb1025e168f385fbac9081b2e4dbb3f7a6c
+        ports:
+          - 8000:8000
+        env:
+          ENABLE_LOGS: true
+          NETWORK: local
+          ENABLE_SOROBAN_RPC: true
+        options: >-
+          --health-cmd "curl --no-progress-meter --fail-with-body -X POST \"http://localhost:8000/soroban/rpc\" -H 'Content-Type: application/json' -d '{\"jsonrpc\":\"2.0\",\"id\":8675309,\"method\":\"getNetwork\"}' && curl --no-progress-meter \"http://localhost:8000/friendbot\" | grep '\"invalid_field\": \"addr\"'"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 50
```

Comment:
The quickstart image fails to run as a service for some reason. Horizon fails to start. See a work around in this issue:
- https://github.com/stellar/quickstart/issues/541

Reply:
This problem with quickstart failing to start as a service will be fixed by:
- https://github.com/stellar/quickstart/pull/540